# $DragonFly: src/secure/usr.sbin/sshd/Attic/auth.c.patch,v 1.2 2004/08/30 21:59:58 geekgod Exp $
--- /home/sullrich/openssh-3.9p1/auth.c	2004-08-19 17:28:41.000000000 +0000
+++ /usr/src/crypto/openssh-3.9p1/auth.c	2004-08-12 12:40:25.000000000 +0000
@@ -472,7 +472,7 @@
 	if (!allowed_user(pw))
 		return (NULL);
 #ifdef HAVE_LOGIN_CAP
-	if ((lc = login_getpwclass(pw)) == NULL) {
+	if ((lc = login_getclass(pw->pw_class)) == NULL) {
 		debug("unable to get login class: %s", user);
 		return (NULL);
 	}
